templates/page_object.rb

```diff
@@ -0,0 +1,7 @@
+class PageObject
```

Comment:
Yes, I agree. I feel like we're arguing, which wasn't my intention - I
think a page_object library with a punny name is a good idea.

I have actually copied this exact code into a lot of projects, though.
Perhaps this deserves its own Trello card?

On Thu, Jun 25, 2015 at 4:07 PM, Joe Ferris <notifications@github.com>
wrote:

> In templates/page_object.rb
> <https://github.com/thoughtbot/suspenders/pull/583#discussion_r33298705>:
>
> > @@ -0,0 +1,7 @@
> > +class PageObject
>
> The experiment wasn't as well-defined in that card as it generally is for
> our newer ones. For example, there isn't a success criteria or an expected
> outcome. The feelings on that card also weren't unanimous, so while I'd
> agree that page objects have proven to be useful, I don't think that we've
> shown as a team that we want to use page objects on all new projects going
> forward.
>
> For projects that do want page objects, I think it would be great to
> define a library with a base class that provides them, rather than copying
> and pasting them into all new projects. To me, the threshold for Suspenders
> is, "will I want to copy this code into every project I make?" I think we
> can find a better way to share this particular piece of code than copying
> and pasting it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/suspenders/pull/583/files#r33298705>.
>



-- 
Gabe Berke-Williams
http://gabebw.com/